reference-architecture/gce-ansible/playbooks/roles/gce-cloudconfig/tasks/main.yaml

```diff
@@ -1,9 +1,9 @@
 ---
-- name: ensure the /etc/orgin folder exists
-  file: name=/etc/origin state=directory
+- name: ensure the /etc/origin/cloudprovider folder exists
+  file: name=/etc/origin/cloudprovider state=directory recurse=yes
 
 - name: configure gce cloud config options
-  ini_file: dest=/etc/origin/gce.ini section=global option={{ item.key }} value={{ item.value }} state=present create=yes
+  ini_file: dest=/etc/origin/cloudprovider/gce.conf section=global option={{ item.key }} value={{ item.value }} state=present create=yes
```

Comment:
I think we will clobber the multizone value in this file with openshift-ansible, which isn't an issue unless you are deploying to only a subset of the zones in the region.